Packet++/header/ProtocolType.h

```diff
@@ -287,6 +287,11 @@ namespace pcpp
 	 */
 	const ProtocolType SomeIP = 0x100000000000;
 
+	/**
+	 * SOME/IP Transport Protocol
+	 */
+	const ProtocolType SomeIPTP = 0x200000000000;

```

Comment:
Now that `SomeIpTpLayer` inherits from `SomeIpLayer` we probably don't need another const here. To get the TP layer one can do: 
```cpp
packet.getLayerOfType<pcpp::SomeIpTpLayer>();
```

If it not `nullptr` it means the packet has a TP layer